### PR TITLE
Establish code base in repository

### LINKED ISSUE #1: Integrate with Global View API
Resolve where development should take place, my understanding is to install mwclient on target machine (TBD) , need to talk to @slifty.

### LINKED ISSUE #2: Via API write Python script to retrieve proposal data

### PR DESCRIPTION
**Pull request by isabelwu6464**
## Description  
This PR establishes a preliminary file structure for the repository. This includes an environment.yml file and requirements.txt file to setup an environment using either conda or pipenv respectively. It also includes a src folder with an initial script integrating with Global View API.

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  - six=1.16.0=pyh6c4a22f_0
+  - sqlite=3.35.5=h8ffe710_0
+  - tzdata=2021a=he74cb21_0
+  - urllib3=1.26.5=pyhd8ed1ab_0
+  - vc=14.2=hb210afc_4
+  - vs2015_runtime=14.28.29325=h5e1d092_4
+  - wheel=0.36.2=pyhd3deb0d_0
+  - win_inet_pton=1.1.0=py39hcbf5309_2
+  - wincertstore=0.2=py39hcbf5309_1006
+prefix: C:\Users\isabe\Anaconda\envs\lfc_analysis_env
```
Comment (slifty) [inline environment.yml]: This line is specific to your local instance and should be removed -- I don't know conda well enough to know what the purpose of this line is, so will rely on you to research it.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,11 @@
 # lfc-analysis
-A repository for analysis tools for LFC + torque + Global View
+A repository for analysis tools for LFC + torque + Global View in order to create geographic visualizations of proposal data.  
+
+# Setup Development Environment  
+To set up the development environment use either:
+```
+pip install -r requirements.txt
+
+conda env create -f environment.yml
```
Comment (slifty) [inline README.md]: We should either use pip or conda -- not both.

File: .gitignore [hunk 3]
```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# Config file for credentials
+config.py
```
Comment (slifty) [inline .gitignore:132]: Could you also provide a template of the config (and update README explaining that it needs to be populated).

Name could be `config.template.py`

**Review by slifty — CHANGES_REQUESTED** [review]
A few tweaks but overall lookin' like a great start.

File: environment.yml (hunk 1 above)
Comment (isabelwu6464) [reply environment.yml]: I have switched to pipenv and this file will no longer be necessary.

File: README.md (hunk 2 above)
Comment (isabelwu6464) [reply README.md]: Replaced with pipenv

File: .gitignore (hunk 3 above)
Comment (isabelwu6464) [reply .gitignore:132]: Added config template file to input API credentials and updated readme.

**Review by slifty — APPROVED** [review]
This is excellent!

Only note for general reference: `config_template.py` I think it is more standard to go with `config.template.py`

You're welcome to keep it as you have it since you've documented it appropriately!

**Pull request merged**